pkgs/servers/sql/postgresql/ext/pg_hint_plan.nix

```
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchFromGitHub, postgresql }:
+
+stdenv.mkDerivation rec {
+  pname = "pg_hint_plan";
+  version = "14-1.4.0";
+
+  src = fetchFromGitHub {
+    owner = "ossc-db";
+    repo = pname;
+    rev = "RE${builtins.replaceStrings ["-" "."] ["_" "_"] version}";
```

I'm sorry, I missed the `L`
```suggestion
    rev = "REL${builtins.replaceStrings ["-" "."] ["_" "_"] version}";
```
Besides that LGTM

---

Oh, I'm sorry, I missed that too!